sonar-db/src/main/java/org/sonar/db/version/VarcharColumnDef.java

```diff
@@ -25,14 +25,20 @@
 
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
-import static org.sonar.db.version.ColumnDefValidation.validateColumnName;
+import static org.sonar.db.version.Validations.validateColumnName;
 
 /**
  * Used to define VARCHAR column
  */
 public class VarcharColumnDef extends AbstractColumnDef {
   public static final int MAX_SIZE = 4_000;
+  /**
+   * @deprecated use {@link #UUID_SIZE} instead
+   */
+  @Deprecated
   public static final int UUID_VARCHAR_SIZE = 50;
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)